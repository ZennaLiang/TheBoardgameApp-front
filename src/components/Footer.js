import React from "react";

const Footer = () => (
  <div className="row footer bg-primary py-3">
    <div className="col-12 text-center">
      Â©2020 TheBoardGameGuru Team. All rights reserved.
    </div>
  </div>
);

export default Footer;
