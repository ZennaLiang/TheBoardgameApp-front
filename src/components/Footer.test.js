import React from "react";
import renderer from "react-test-renderer";
import Footer from "./Footer";

describe("<Footer />", () => {
  it("render Footer", () => {
    const tree = renderer.create(<Footer />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
