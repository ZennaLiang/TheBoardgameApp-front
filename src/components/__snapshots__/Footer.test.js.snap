// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> render Footer 1`] = `
<div
  className="row footer bg-primary py-3"
>
  <div
    className="col-12 text-center"
  >
    Â©2020 TheBoardGameGuru Team. All rights reserved.
  </div>
</div>
`;
